aldryn_newsblog/models.py

```diff
@@ -175,6 +176,6 @@ def __str__(self):
         return u'Latest entries: {0}'.format(self.latest_entries)
 
     def get_articles(self):
-        articles = Article.objects.active_translations(get_language()).filter(
-            app_config=self.app_config)
+        articles = Article.published.active_translations(
+            get_language()).filter(app_config=self.app_config)
```

Comment:
You have tested if only `active_translations`  is enough to return object in language that is not the default? Parler docs say to use `language` first, ex:

```python
language = get_language()
articles = Article.objects.published().language(language).active_translations(language).filter(app_config=self.app_config)
```

Also, I would prefer to pass language as argumento to get_articles so I can pass correct language used in view because I get it with `get_language_from_request`.